tutorials/vue/resize.md

```diff
@@ -0,0 +1,130 @@
+To create a resize event, you have a couple options.
+
+## Add an Event Listener
+
+This is the simpler approach of the two. You must utilize the
+`created()` and `destroyed()` lifecycle hooks to invoke the `addEventListener()`
+and `removeEventListener()` functions respectively.
+
+```javascript
+const app = new Vue({
+    data: function() {
+        return {
+            width: 0,
+            height: 0
+        }
+    },
+    created() {
+        window.addEventListener('resize', this.getDimensions);
+    },
+    destroyed() {
+        window.removeEventListener('resize', this.getDimensions);
+    },
+    methods: {
+        getDimensions() {
+            this.width = document.documentElement.clientWidth;
+            this.height = document.documentElement.clientHeight;
+        }
+    }
+})
+```
+
+The downside is that is would only be local to the component you are using it in. If you need something
+that is universally accessible then the next option is for you.
+
+## Create a Vue Directive
+
+Similar to [creating a custom scroll event](/tutorials/vue/scroll), you can create a directive for window resizing.
+
+```javascript
+Vue.directive('resize', {
+    inserted: function(el, binding) {
+        const onResizeCallback = binding.value
+        window.addEventListener('resize', () => onResizeCallback());
+    }
+});
+
+const app = new Vue({
```

Comment:
Add an example of using the resize directive - I think you need to use it in a template

Reply:
I use it in the second example.

Reply:
I meant this Vue instance doesn't have a `template`, which makes it tricky to see how to use this directive.